src/routes/app/streams/+page.svelte

```diff
@@ -124,12 +124,11 @@
 </script>
 
 <svelte:head>
-  <title>Dashboard | Drips</title>
+  <title>Streams | Drips</title>
```

Comment:
Should this be dynamic depending on the current page (streams vs. crowdfunding)?